### DISCUSSION TITLE
support for 10.0.5.48069?

### DISCUSSION THREAD

**Discussion by robwetz** [trickerer/Trinity-Bots#361] [General]
@trickerer, apologies if this question has already been asked (i couldn't find it if it has).  are there plans to support client 10.0.5.48069 of tc?  thanks!

**Comment by trickerer** [comment]
Sorry, no. Nothing above 3.x.x will ever be supported - I can't afford supporting an expansion I never played.

**Reply by robwetz** [reply]
makes sense.  thanks for getting back to me!

**Reply by Aztec2012** [reply]
You can play it on Retail-Core project if you want.
